manifests/m/Microsoft/WindowsAppSDKRuntime/1.0/Microsoft.WindowsAppSDKRuntime.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsAppSDKRuntime
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: Windows App SDK Runtime
+License: MIT License
+ShortDescription: 'Includes all the MSIX packages required by an app to be able to use the Windows App SDK. These packages include: Framework, Main, and DDLM.'
```

The installer will install Singleton package also.

---

Yes, we should change to:
```suggestion
ShortDescription: 'Includes all the MSIX packages required by an app to be able to use the Windows App SDK. These packages include: Framework, Main, Singleton, and DDLM.'
```